src/BaseTemplates/WebAPI/project.json

```diff
@@ -36,7 +36,10 @@ $if$ ($context$ == WebCore) $else$
   },
 $endif$
   "tools": {
-    "Microsoft.AspNetCore.Server.IISIntegration.Tools": "1.0.0-*"
+    "Microsoft.AspNetCore.Server.IISIntegration.Tools": {
+      "version": "__AspNetCorePlatformVersion__",
+      "imports": "portable-net45+wp80+win8+wpa81+dnxcore50"
```

Comment:
I was going by what was mentioned in https://github.com/aspnet/Announcements/issues/169 for this tool.

@moozzyk can we reduce these down to what Taylor suggests?